Monika After Story/game/script-topics.rpy

```diff
@@ -4940,3 +4940,299 @@ label monika_asks_family:
                 m 1f "You can tell me about your family when you're ready, [player]." 
                 m 1j "I love you very much!"                              
     return
+    
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_fun_facts_open",category=['misc'],prompt="Fun facts",pool=True))
+
+default fun_facts_started = False
+default persistent._mas_funfactfun = True
+define mas_funfact.fun_count = 20
+define mas_funfact.bad_count = 4
+
+label monika_fun_facts_open:
+    #Determines the fact number and whether it's a bad fact
+    fun_facts_bad_chance = renpy.random.randint(1,100)
```

Comment:
you need the `$` character before `fun_facts_bad_chance` otherwise it crashes the mod 

The line should look like this:
`    $ fun_facts_bad_chance = renpy.random.randint(1,100)`